packer/endpointserver.yaml

```diff
@@ -0,0 +1,88 @@
+builders:
+  - type: amazon-ebs
+    ami_name: Packer build for endpointserver 8
+    #get a bit more compute, we are compiling stuff
+    instance_type: t2.large
+    source_ami: ami-c90195b0
+    ssh_username: ec2-user
+#built 1st version at ami-7ca0cc05. need to check how to install memcached
+#up-to-date ami at ami-f39af68a
+provisioners:
+  - type: file
+    source: packer/files/nginx.conf
+    destination: /tmp/nginx.conf
+    
+  - type: file
+    source: packer/files/www.conf
+    destination: /tmp/www.conf
+
+  - type: shell
+    inline:
+      #needed to ensure that awscli actually installs
+      #https://bugzilla.redhat.com/show_bug.cgi?id=1417304
+      - sudo yum-config-manager --enable rhui-REGION-rhel-server-optional
+      - sudo yum -y update
+      - sudo rpm -ivh https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+
+      - sudo yum -y install epel-release
+      - sudo yum -y groupinstall 'Development Tools'
+      - sudo yum -y install bzip2 python-docutils nginx awscli mysql-devel libmemcached-devel memcached-devel libxml2-devel vim curl-devel openssl-devel mlocate curl libmcrypt libmcrypt-devel
+      - sudo mv /tmp/nginx.conf /etc/nginx/nginx.conf
+      
+      #add in filebeat installation here. config will be done in userdata.
+      - sudo rpm -Uvh https://artifacts.elastic.co/downloads/beats/filebeat/filebeat-6.1.3-x86_64.rpm
+      - sudo systemctl disable filebeat
+      #FIXME: does it work on this version???
+      #- curl http://ie1.php.net/distributions/php-7.2.2.tar.bz2 > php-7.2.2.tar.bz2
+      #- tar xvjf php-7.2.2.tar.bz2
+      #- cd php-7.2.2
+      #- ./configure --with-mysql --with-zlib --with-mysqli=/usr/bin/mysql_config --with-openssl --with-curl --enable-dba --with-mcrypt --enable-ssl
+      #- make -j2
+      #- make test
+      #- sudo make install
+      #- cd ..
+      #- rm -rf php-7.2.2*
+
+      - sudo rpm -Uvh http://rpms.famillecollet.com/enterprise/remi-release-7.rpm
+
+      - echo ------------------------------------
+      - echo Installing PHP
+      - echo ------------------------------------
+      
+      - sudo yum -y --enablerepo=remi,remi-php72 install php php-common php-mysql php-fpm php-pecl-memcache php-pecl-memcached
```

Comment:
how trustworthy is this rpm source do you think? how up-to-date is it kept?

Reply:
It looks pretty trustworthy to me. It was last updated on 22/6/2018.